buildingmotif/dataclasses/template.py

```diff
@@ -275,6 +319,8 @@ def inline_dependencies(self) -> "Template":
         # start with this template's parameters; this recurses into each
         # dependency to inline dependencies
         for dep in self.get_dependencies():
+            if dep.template is None:
```

Comment:
In these cases I'm not sure if we should throw a `TemplateNotFound` or pass like this. @gtfierro thoughts?

Reply:
Elsewhere, we use a `error_on_missing_imports` flag on the compile method to determine whether or not missing libraries should raise an error or just be ignored. We could do a similar flag here? something like `error_on_missing_dependencies` that defaults to True but you can disable that behavior if you want. We'd have to provide access to that flag in `Library.load` and perhaps a couple other entrypoints to creating templates

Reply:
How is this handled now? It looks like there is still a "continue" in there if dep.template is None. It would be nice to get a comment in the code explaining what's going on here, or even something in the doc string?

Reply:
ok. It looks like the review was on an old version of the code. This is currently throwy without an option as attempting to inline when not all dependencies exist seems bad.

Reply:
Thanks! That looks like the right behavior